api/bonsai_api/crud/sample.py

```diff
@@ -303,20 +294,27 @@ async def get_samples_summary(
         },
     )
 
-    # query database
-    cursor = db.sample_collection.aggregate(pipeline)
-    # get query results from the database
-    results = await cursor.to_list(None)
-    result = results[0]
-
-    return MultipleRecordsResponseModel(
-        data=result["data"],
-        records_total=(
-            0
-            if len(result["records_total"]) == 0
-            else result["records_total"][0]["count"]
-        ),
-    )
+    if isinstance(include_samples, list) and len(include_samples) == 0:
```

Comment:
I am curious about what made you do `isinstance` here instead of `include_samples is not None`